data/remnant outfits.txt

```diff
@@ -465,6 +465,19 @@ outfit "Tuning Rifle"
 	"unplunderable" 1
 	description "The Tuning Rifle uses sound waves to destabilize the weaker internal tissue of many races and liquefies them."
 
+outfit "Void Rifle"
+	category "Special"
+	licenses
+		Remnant
+	cost 1000000
+	thumbnail "outfit/void rifle"
+	"capture attack" 20
+	"capture defense" 20
+	"energy consumption" 0.017
+	"unplunderable" 1
+	description "When fired, the Void Rifle emits a form of radiation that has a strong paralyzing effect on the nervous system. With an area of effect that penetrates walls and other solid objects, it is an incredibly effective weapon."
+	description "The Remnant made a significant number of these weapons during the early years of their research on the void sprites, but have not been able to make any new ones since the Archon stopped their harvests. Without a supply of new tentacles to use, the Remnant have been unable to manufacture more. While durable, they do rarely last more than a decade, so even the best cared for of the old Void Rifles degraded into uselessness centuries ago. These highly effective weapons are credited with helping the Remnant safeguard their refuges from ground-based threats during their early years."
```

Comment:
Switched the order of the paragraphs, then indented the second one.

In terms of emotion, I realize that in general putting emotion into the character isn't the best thing to do. If the game could handle putting someone in a stressful situation like that with a custscene decision, that would be so much better. But since we can't, I think this works. 

On the flip side, sometimes (used sparingly), I think it is good to give emotions to the player, otherwise the game ends up being too impersonal.